src/datastructures.apib

```diff
@@ -140,3 +140,121 @@
         + timeTracking.deleted
         + timeTracking.updated
         + user.deactivated
+
+## WorkOrderData (object)
++ customer (object, required)
+    + type (enum[string], required)
+        + Members
+            + contact
+            + company
+    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
++ datetime: `2017-05-26T10:01:49+00:00` (string, required)
++ description (string, optional)
++ department_id: `cef01135-7e51-4f6f-a6eb-6e5e5a885ac7` (string, optional)
++ layout_id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string, optional)
++ milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
++ event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
++ intervention_address (object, optional)
+    + One Of
+        + customer (object, required)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + address (Address)
++ distance: `25.2` (number, optional)
++ users (array, required)
+    + (object)
+        + user_id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string, required)
+        + hourly_rate: `65` (number, optional)
+        + work_hours (array, optional)
+            + (object)
+                + One Of
+                    + Properties
+                        + started_at: `2017-04-26T10:01:49+00:00` (string, required)
+                        + duration: 3600 (number, required) - In seconds
+                    + Properties
+                        + started_at: `2017-04-26T10:01:49+00:00` (string, required)
+                        + ended_at: `2017-04-26T16:02:00+00:00` (string, required)
+                + description (string, optional)
++ line_items (array, optional)
+    + (object)
+        + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
+        + quantity: `3` (number, optional)
+        + description: `An awesome product` (string, optional)
+        + unit_price (object, optional)
```

Comment:
Q : I'm not sure if this API should care about prices?

Reply:
Same actually for descriptions etc

Reply:
It's possible a custom price is being used for an existing product. E.g. when a contractor is getting the product on short notice from a different supplier.

Reply:
Agreed to remove this from the proposal for now. Will be revisited later.